core/src/navigation/LeftNav.html

```diff
@@ -231,24 +246,46 @@
     -webkit-box-shadow: 1px 1px 2px 0 rgba(0, 0, 0, 0.05);
     box-shadow: 1px 1px 2px 0 rgba(0, 0, 0, 0.05);
   }
-  .fd-side-nav__link {
-    .sap-icon--action {
-      //external-link icon
-      margin-left: 0.3em;
-      vertical-align: middle;
-    }
-    .fd-side-nav__icon {
-      width: 2em;
+
+  .fd-side-nav {
+    min-height: 100%;
+    margin-bottom: calc(-1 * #{$footerHeight});
+
+    &::after {
+      content: '';
+      display: block;
+      height: $footerHeight;
     }
-  }
-  .fd-side-nav__sublink {
-    &.fd-side-nav__link--icon {
-      padding-left: 16px;
+
+    &__link {
+      .sap-icon--action {
+        //external-link icon
+        margin-left: 0.3em;
+        vertical-align: middle;
+      }
+      .fd-side-nav__icon {
+        width: 2em;
+      }
     }
-    .fd-side-nav__icon {
-      width: 2em;
+
+    &__sublink {
+      &.fd-side-nav__link--icon {
+        padding-left: 16px;
+      }
+      .fd-side-nav__icon {
+        width: 2em;
+      }
     }
   }
+
+  .lui-sideNavFooter {
+    display: inline-block;
+    width: 100%;
+    color: var(--fd-color-text-4);
+    /* height: $footerHeight; */
+    padding: 13px 22px;
```

Comment:
```suggestion
    padding: $side-nav-footer--padding 20px;
```

Let's use a variable here. And the navigation items have 20px padding. I think it's better to have the same here.